.golangci.yml

```diff
@@ -33,3 +31,5 @@ issues:
     - G103 # Use of unsafe calls should be audited
     - G204 # Subprocess launched with variable
     - G304 # Potential file inclusion via variable
+    - G306 # WriteFile permissions 0600 or less to be audited
+    - G307 # Deferring unsafe method "Close" on type "*os.File" to be audited
```

Comment:
I think each these two groups of errors should be handled separately outside this change. Each uncovered numerous issues which were not trivial to resolve or was not surgical like the other fixes in this PR.